<ui version="4.0" >
 <class>VeritasConfig</class>
 <widget class="QWidget" name="VeritasConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>497</height>
   </rect>
  </property>
  <property name="whatsThis" >
   <string>Test settings.

Select the test framework used in the project and set its detailed configuration.

Then you can add, edit and remove the test executables to be run. Once added, they can be run in the tool view of the framework test.

Note that you may not be able to add, edit or remove tests if the test framework and the build manager support automatic test discovery. In that case, just select the project in the tool view of the framework test and the test executables will be added automatically.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QFrame" name="frameworkPanel" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="frameworkLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="KComboBox" name="frameworkSelection" >
          <property name="toolTip" >
           <string>Test framework to use</string>
          </property>
          <property name="whatsThis" >
           <string>Select the test framework used in this project.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addExecutable" >
          <property name="toolTip" >
           <string>Add a new test executable</string>
          </property>
          <property name="whatsThis" >
           <string>Add a new test executable to be run.
The executable must use the specified framework. 

Note that you may not be able to add a new test if the test framework and the build manager support automatic test discovery. In that case, just select the project in the tool view of the framework test and the test executables will be added automatically.</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="expandDetails" >
          <property name="toolTip" >
           <string>Expand framework details</string>
          </property>
          <property name="whatsThis" >
           <string>Expand the framework configuration details.</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="executables" >
     <property name="whatsThis" >
      <string>The list of test executables for the selected framework.

The list may be disabled if the test framework and the build manager support automatic test discovery.</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="executableFieldsLayout" >
      <item>
       <widget class="QScrollArea" name="executablesScroll" >
        <property name="mouseTracking" >
         <bool>false</bool>
        </property>
        <property name="widgetResizable" >
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>553</width>
           <height>410</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QVBoxLayout" name="scrollExeLayout" >
            <item>
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>216</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
